network/stream/stream.go

```diff
@@ -92,6 +92,9 @@ var (
 			WantedHashes{},
 		},
 	}
+
+	// pause the msgHandler execution, used only for tests
+	MessagePauser protocols.TestMsgPauser = nil
```

Comment:
MessagePauser does not have to be exported. Maybe even to use the original name handleMsgPauser?